File: app/src/main/java/com/imceits/aungtuntun/alephcodeassignment/data/entity/Friends.java
package com.imceits.aungtuntun.alephcodeassignment.data.entity;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.Ignore;
import androidx.room.PrimaryKey;

@Entity(tableName = "Friends")
public class Friends {
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "friend_id")
    private int id;
    @ColumnInfo(name = "name")
    private String name;

    public Friends() {
    }

    @Ignore
    public Friends(String name) {
        this.name = name;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/com/imceits/aungtuntun/toolsbox/data/entity/Friends.kt
package com.imceits.aungtuntun.toolsbox.data.entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "Friends")
data class Friends(
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "friend_id")
    val id: Int,
    @ColumnInfo(name = "name")
    val name: String
)